lib/react-contenteditable.js

```diff
@@ -37,14 +35,15 @@ var ContentEditable = (function (_React$Component) {
     value: function render() {
       var _this2 = this;
 
-      return _react2.default.createElement('div', _extends({}, this.props, {
+      return _react2.default.createElement(this.props.tagName || 'div', { ...this.props,
```

Comment:
Ups! Looks like it didn't work.

But I think this is because the syntax that I praised [is not part of the standard](http://www.ecma-international.org/ecma-262/6.0/#sec-object-initializer), so babel has removed support for it. My bad again.